src/PackagerNewCommand.php

```diff
@@ -156,9 +161,9 @@ public function handle()
 
     protected function interactiveReplace($vendor, $name, $fullPath)
     {
-        $author = $this->ask('Who is the author?');
-        $authorEmail = $this->ask('What is the author\'s e-mail?');
-        $authorSite = $this->ask('What is the author\'s website?');
+        $author = $this->ask('Who is the author?', \Config::get('packager.author'));
+        $authorEmail = $this->ask('What is the author\'s e-mail?', \Config::get('packager.author_email'));
+        $authorSite = $this->ask('What is the author\'s website?', \Config::get('packager.author_site'));
         $description = $this->ask('How would you describe the package?');
         $license = $this->ask('Under which license will it be released?');
```

Comment:
I think people also tend to use the same license for all their packages, so this might be in the config too.

Reply:
Done